local_planner/include/local_planner/local_footstep_planner.h

```diff
@@ -240,6 +240,129 @@ class LocalFootstepPlanner {
     terrain_grid_ = grid_map;
   }
 
+  // Compute swing apex by ground and hip height
+  inline double computeSwingApex(int leg_idx, const Eigen::VectorXd &body_plan,
```

Comment:
Any reason these functions should be in the header file?

Reply:
Normally inlines are used for functions with small definitions to save time accessing the implementation of the function when it is called. I think these can be moved to cpp.

Reply:
Yeah ideally inlines should be only a few lines or smaller. We may have gotten a bit aggressive in other parts of the code...